### ISSUE TITLE
Keyboard navigation is missing

### ISSUE THREAD

**Issue by rahul286** [HrithikDark/The_WP_Slideshow#3]
Point 5 from https://rtcamp.com/assignments/front-end-engineer/

> The slideshow must support keyboard navigation. The right arrow key to move to the next slide and left arrow key to move to the previous slide.
